tests/FormatTest.php

```diff
@@ -17,7 +17,7 @@
 it('can determine the age in days', function () {
     Carbon::setTestNow(Carbon::create(2016, 1, 1)->startOfDay());
 
-    expect(Format::ageInDays(Carbon::now()))->toEqual('0.00 (1 second ago)');
+    expect(Format::ageInDays(Carbon::now()->subSeconds(5)))->toEqual('0.00 (5 seconds ago)');
```

Comment:
Updated the assertion here to be compatible with Carbon v2 and v3. 
Internally v3 uses floats and no longer integers.

This led to issues when in the diff was ~0 seconds. See [test run](https://github.com/stefanzweifel/laravel-backup/actions/runs/7786191155/job/21230440647#step:7:184).

There's also a graet summary of the major changes in this comment: https://github.com/laravel/framework/pull/49764#issuecomment-1904438086